TodoLogic/src/components/Todo.tsx

```diff
@@ -0,0 +1,76 @@
+import TodoList from "./TodoList";
+import TodoDate from "./TodoDate";
+import "./App.css";
+import { useState, useEffect, useCallback } from "react";
+
+const Todo = (): JSX.Element => {
+  const [showLabel, setShowLabel] = useState(false);
+  const [btnVisible, setBtnVisible] = useState(true);
+  const [data, setData] = useState("");
+
+  const submitHandler = useCallback((e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    let date = new Date();
+    let currdate = date.toLocaleDateString();
+    localStorage.setItem("currdate", currdate);
+  }, []);
+
+  const clickHandler = () => {
+    setShowLabel(!showLabel);
+    setBtnVisible(!btnVisible);
+  };
+
+  const handleKeyUp = useCallback(
+    (event: KeyboardEvent) => {
+      if (event.key === "Enter") {
```

Comment:
This condition is not required. `<form>` tag automatically enables form submission on enter key press.

Reply:
in my case it doesn't work